python_modules/dagster/dagster/core/storage/pipeline_run.py

```diff
@@ -470,21 +493,29 @@ def __new__(
             created_before=check.opt_inst_param(created_before, "created_before", datetime),
         )
 
+    @property
+    def pipeline_name(self):
+        return self.job_name
+
     @staticmethod
     def for_schedule(schedule):
-        return PipelineRunsFilter(tags=PipelineRun.tags_for_schedule(schedule))
+        return RunsFilter(tags=PipelineRun.tags_for_schedule(schedule))
 
     @staticmethod
     def for_partition(partition_set, partition):
-        return PipelineRunsFilter(tags=PipelineRun.tags_for_partition_set(partition_set, partition))
+        return RunsFilter(tags=PipelineRun.tags_for_partition_set(partition_set, partition))
 
     @staticmethod
     def for_sensor(sensor):
-        return PipelineRunsFilter(tags=PipelineRun.tags_for_sensor(sensor))
+        return RunsFilter(tags=PipelineRun.tags_for_sensor(sensor))
 
     @staticmethod
     def for_backfill(backfill_id):
-        return PipelineRunsFilter(tags=PipelineRun.tags_for_backfill_id(backfill_id))
+        return RunsFilter(tags=PipelineRun.tags_for_backfill_id(backfill_id))
+
+
+register_serdes_tuple_fallbacks({"PipelineRunsFilter": RunsFilter})
+PipelineRunsFilter = RunsFilter
```

Comment:
[1]

Reply:
my b

Reply:
lets at least leave a comment here if you are not going to include active deprecation warning